sosw/components/helpers.py

```diff
@@ -1060,3 +1062,31 @@ def unwrap_event_recursively(event: Dict, sources: Optional[List[str]] = None) -
             break
 
     return messages
+
+
+def small_int_from_string(input_string: str, num_digits: int = 2) -> int:
+    """
+    Generate a small integer based on the input string using its MD5 hash.
```

Comment:
Could be useful for example if you plan to use this custom number for some kind of partitioning or unsorted batching.